src/platforms/platform.dom.js

```diff
@@ -162,23 +128,24 @@ module.exports = function(Chart) {
 			}
 		};
 
-		// Let's keep track of this added iframe and thus avoid DOM query when removing it.
-		stub.resizer = createResizer(notify);
+		if (!erd) {
+			erd = elementResizeDetectorMaker({
+				strategy: 'scroll',
+			});
+		}
 
-		node.insertBefore(stub.resizer, node.firstChild);
+		erd.listenTo(node, notify);
+		stub.resizer = true;
```

Comment:
This property is used to check if the resizer has been initialized for this particular chart. It's used for the same purpose like before, to check if resizer is still initialized when running the deferred resize callback. It's removed when `removeResizeListener` is called.